po/ru.po

```diff
@@ -1647,28 +1771,64 @@ msgid ""
 "      * It is also extensible with sub command plugins as well (such as [cargo clippy]).\n"
 "  * Read more from the [official Cargo Book]"
 msgstr ""
+"* У Rust стремительный график релизов: новая версия выходит каждые шесть недель.\n"
+"Новые версии поддерживают обратную совместимость со старыми версиями ---\n"
+"в дополнение они предоставляют новые функциональные возможности.\n"
+"\n"
+"* Существует три вида релизов: ”stable”, ”beta” и ”nightly”.\n"
+"\n"
+"* Новые функции тестируются на ”nightly”, ”beta” --- это то, что становится\n"
+"”stable” каждые шесть недель.\n"
+"\n"
+"* У Rust также есть [редакции]: текущая редакция --- Rust 2021. Предыдущими\n"
+"редакциями были Rust 2015 и Rust 2018.\n"
+"\n"
+"   * Редакциями позволено вносить обратно--несовместимые изменения в язык.\n"
+"\n"
+"  * Чтобы избежать сбоев, редакцию для своего пакета можно явно указать в файле `Cargo.toml`.\n"
+"\n"
+"  * Чтобы избежать разделения экосистемы, компилятор Rust может смешивать код,\n"
+"написанный для разных редакций.\n"
+"\n"
+"  * Стоит упомянуть, что использование компилятора напрямую, а не через `cargo`, является редким явлением\n"
+"(большинство пользователей никогда этого не делают).\n"
+"\n"
+"  * Cargo сам по себе является чрезвычайно\n"
+"мощным и всеобъемлющим инструментом. Он поддерживает множество дополнительных функций, включая, помимо прочего:\n"
+"      * Project/package structure\n"
+"      * [workspaces]\n"
+"      * Dev Dependencies and Runtime Dependency management/caching\n"
+"      * [build scripting]\n"
```

Comment:
Перевод пропущен?
Без понятия как работают ссылки определенные ниже по тексту с переводом но с "Project/package structure" и "Dev Dependencies and Runtime Dependency management/caching" проблем быть не должно.

Reply:
Yeah, I've skipped this one on purpose since was not sure about the `Dev Dependencies and Runtime Dependency management/caching`, especially about the `caching` part. 
IMO It's ok to omit the `caching` word, since it doesn't contribute much. One possible translation would be `Управление зависимостями разработки (dev) и времени выполнения (runtime)`

Reply:
Hey folks, just a small comment: as translators, you're in a unique position to help fix any awkwardness in the English sentences. Most people just skim through the course, but you'll actually be reading the whole thing :slightly_smiling_face: 

So you're invited to send small PRs to make the sentences easier to read and understand. Those changes will then slowly make their way into the translations and hopefully improve things for everybody.

Reply:
Управление зависимостями времени разработки (dev) и времени выполнения (runtime)

Reply:
ну в российской индустрии так говорят. собственно чем это хуже, чем времени исполнения.  и то и то с точки зрения нормы литературного русского языка звучит неправильно.
если нет, то с для лучше, чем без для.

Reply:
👌